frontend/src/components/forms/EditLog.tsx

```diff
@@ -375,7 +375,7 @@ const EditLog = ({
                       placeholder="Select Residents"
                       onChange={handleResidentsChange}
                       defaultValue={residentOptions.filter(
-                        (item) => logRecord.residents.includes(item.label),
+                        (item) => logRecord.residents && logRecord.residents.includes(item.label),
```

Comment:
is there some sort of error that appears that required this to be changed? residents should never be null